hack/build-image.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+OS_ROOT=$(dirname ${BASH_SOURCE})/..
+
+# Register function to be called on EXIT to remove generated binary.
+function cleanup {
+  rm "${OS_ROOT}/images/origin-web-console-server/bin/origin-web-console-server"
+}
+trap cleanup EXIT
+
+cp -v ${OS_ROOT}/_output/bin/origin-web-console-server "${OS_ROOT}/images/origin-web-console-server/bin/origin-web-console-server"
```

Comment:
Do we need to make sure the directory exists?

```
mkdir -p "${OS_ROOT}/images/origin-web-console-server/bin"
```

Reply:
done